views/sis/index.js

```diff
@@ -5,20 +5,66 @@
  */
 
 import React from 'react'
-import {StyleSheet} from 'react-native'
+import {
+  StyleSheet,
+  Text,
+  View,
+  Navigator,
+} from 'react-native'
 
 import TabbedView from '../components/tabbed-view'
+import Button from 'react-native-button'
 import tabs from './tabs'
-
+import {loadLoginCredentials} from '../../lib/login'
```

Comment:
loadLoginCredentials is an asynchronous function, right?

So it'll return a promise.